src/styles/index.scss

```diff
@@ -1,3 +1,17 @@
-body {
+@use './header.scss';
+@use './main.scss';
+
+.page {
+  display: flex;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  scroll-behavior: smooth;
+}
```

Comment:
The `.page` class seems to be applied to the `body` or `html` element. It's better to apply these styles directly to the `body` or `html` tag. It's a common practice and it increases code readability.